### ISSUE TITLE
Add option --quiet

### ISSUE THREAD

**Issue by evilebottnawi** [JakubOnderka/PHP-Parallel-Lint#73]
For suppress output, but throw error if found.

``` shell
..............................................  60/839 (7 %)
............................................................ 120/839 (14 %)
............................................................ 180/839 (21 %)
............................................................ 240/839 (28 %)
............................................................ 300/839 (35 %)
............................................................ 360/839 (42 %)
............................................................ 420/839 (50 %)
............................................................ 480/839 (57 %)
............................................................ 540/839 (64 %)
............................................................ 600/839 (71 %)
............................................................ 660/839 (78 %)
............................................................ 720/839 (85 %)
............................................................ 780/839 (92 %)
...........................................................  839/839 (100 %)
```

**Comment by JakubOnderka** [comment]
@evilebottnawi What do you mean that is expected behavior if no error is found? Suppress the whole progress bar and show only `No syntax error found` or even without this message and return only status code `0`?

**Comment by evilebottnawi** [comment]
@JakubOnderka yep

**Comment by MetalArend** [comment]
It would be very welcome to be able to disable the progress bar. Is there a reason you closed this without it being resolved, @evilebottnawi ?

**Comment by evilebottnawi** [comment]
@MetalArend don't using this project now and i have a lot of issue in github, sometimes i close inactive issues :sob:

**Comment by grogy** [comment]
When the issue is up to date for you ( @MetalArend ) then you can open it now. We can resolve it together.

**Comment by MetalArend** [comment]
@grogy Should I best create a new issue?

**Comment by grogy** [comment]
@MetalArend yes, do it please, when it is important for you ;-)
